agent/server_test.go

```diff
@@ -657,6 +657,10 @@ func (r *testSerfClient) Join(peers []string, replay bool) (int, error) {
 	return 0, nil
 }
 
+func (r *testSerfClient) GetCoordinate(node string) (coord *coordinate.Coordinate, err error) {
```

Comment:
I don't quite understand the reason for both testSerfClient and MockFakeSerfClient - can there be just one for testing?